frontend/src/concepts/pipelines/content/pipelinesDetails/pipelineRun/runLogs/LogsTab.tsx

```diff
@@ -126,126 +172,162 @@ const LogsTabForPodName: React.FC<{ podName: string }> = ({ podName }) => {
-              {isSmallScreen() ? null : (
-                <SplitItem>
-                  <Bullseye>
-                    <TextContent>
-                      <TextListItem component={TextListItemVariants.dt}>Step</TextListItem>
-                    </TextContent>
-                  </Bullseye>
-                </SplitItem>
-              )}
-              <SplitItem>
-                <SimpleDropdownSelect
-                  isDisabled={podContainers.length === 0}
-                  options={podContainers.map((container) => ({
-                    key: container.name,
-                    label: container.name,
-                  }))}
-                  value={selectedContainer?.name ?? ''}
-                  placeholder="Select container..."
-                  onChange={(v) => {
-                    setSelectedContainer(podContainers.find((c) => c.name === v) ?? null);
-                  }}
-                  width={150}
-                />
-              </SplitItem>
-            </Split>
-          </SplitItem>
-          <SplitItem>
-            {!podStatus?.completed && (
-              <Button
-                variant={!logsLoaded ? 'plain' : isPaused ? 'plain' : 'link'}
-                onClick={() => setIsPaused(!isPaused)}
-                isDisabled={!!error}
-              >
-                {error ? (
-                  'Error loading logs'
-                ) : !logsLoaded || podStatus?.podInitializing ? (
-                  <>
-                    <Spinner size="sm" /> {isSmallScreen() ? 'Loading' : 'Loading log'}
-                  </>
-                ) : isPaused ? (
-                  <>
-                    <PlayIcon /> {isSmallScreen() ? 'Resume' : 'Resume refreshing'}
-                  </>
-                ) : (
-                  <>
-                    <PauseIcon /> {isSmallScreen() ? 'Pause' : 'Pause refreshing'}
-                  </>
-                )}
-              </Button>
-            )}
-          </SplitItem>
-          <SplitItem isFilled style={{ textAlign: 'right' }}>
-            {downloading ? (
-              <>
-                <Spinner size="sm" />{' '}
-              </>
-            ) : null}
-            {podContainers.length !== 0 ? (
-              <Dropdown
-                toggle={
-                  isSmallScreen() ? (
-                    <KebabToggle
-                      onToggle={() => setIsDownloadDropdownOpen(!isDownloadDropdownOpen)}
-                    />
-                  ) : (
-                    <DropdownToggle
-                      id="download-steps-logs-toggle"
-                      onToggle={() => setIsDownloadDropdownOpen(!isDownloadDropdownOpen)}
+      <StackItem isFilled id="complex-toolbar-demo">
+        <DashboardLogViewer
+          data={data}
+          logViewerRef={logViewerRef}
+          toolbar={
+            <Toolbar className="pf-v5-u-p-sm">
+              <ToolbarContent>
+                <Split hasGutter isWrappable style={{ width: '100%' }}>
```

Comment:
Please use `ToolbarGroup` and `ToolbarItem` as it suggests here https://www.patternfly.org/components/toolbar#with-groups-of-items instead of creating the `Split` layout. The toolbar layout could be broken when the window size shrinks a little bit.

<img width="574" alt="Screenshot 2023-12-11 at 3 37 17 PM" src="https://github.com/opendatahub-io/odh-dashboard/assets/37624318/f731ec13-53eb-46fe-b8c9-0ae40a71544a">

Reply:
I have tried using the `ToolbarGroup` and `ToolbarItem`, but the toolbar seems to overflow, when the drawer is at it's minSize.
[Screencast from 2023-12-12 19-37-22.webm](https://github.com/opendatahub-io/odh-dashboard/assets/99261071/86ab3ea3-3dc5-4716-826c-0ae5748a6c1c)

Reply:
https://www.patternfly.org/components/overflow-menu#group-types Overflow menu should be the component you are looking for. When the drawer shrinks to a small size, these options should be put into a toggle dropdown

Reply:
great!! I wasn't aware of this component. Thanks for sharing this.

Reply:
I have tried using overflow menu and `ToolbarToggleGroup`. This will work only when the window size resizes to small size, not the drawer.

Reply:
There is `breakpointReference` prop to set the container you want to listen to, you can set it to the drawer panel. https://www.patternfly.org/components/overflow-menu/#overflowmenu

Reply:
@DaoDaoNoCode because of the `isSmallScreen()` I'm getting `KebabToggle` inside `kebabToggle`. And the toolbar layout is likely change or will have more options. conversation is going on in slack https://redhat-internal.slack.com/archives/C05SMJ09DD2/p1703062044335869?thread_ts=1701764216.953159&cid=C05SMJ09DD2
You want me to address this layout changes in this PR after finalizing the design or will have layout issues in different PR?

Reply:
We can do that in a different PR